minio/base/service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: minio
+spec:
+  selector:
+    app: minio
```

Comment:
Remove this; let kustomize manage selectors for you.

Reply:
Sounds good, I made this change.